doc/man3/EVP_DigestSignInit.pod

```diff
@@ -31,6 +31,10 @@ B<*pctx> is overwritten. The EVP_PKEY_CTX value returned must not be freed
 directly by the application (it will be freed automatically when the EVP_MD_CTX
```

Comment:
I will try to change the wording. But by the way, how can you put a comment outside the 'diff' (green) zones in a patch on Github?

Reply:
It will let you put a comment anywhere that it shows to you - even if it isn't green/red. It only shows you a limited amount of context though, so if you want to put a comment outside the white context lines you're out of luck.